### PR TITLE
Add/run groups to cli

### PR DESCRIPTION
**Pull request by mathnogueira**
This PR...

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Loom video

Add your loom video here if your work can be visualized

### CODE REVIEW

**Comment by xoscar** [comment]
Video
https://www.loom.com/share/e3ad48454ba94fdfa5d4a21ba0fbd0a9

File: cli/cmd/resource_run_cmd.go [hunk 1]
```diff
@@ -5,75 +5,48 @@ import (
...
+			return runSingleFile(ctx)
 
 		}, runParams),
 		PostRun: teardownCommand,
 	}
 
-	runCmd.Flags().StringVarP(&runParams.DefinitionFile, "file", "f", "", "path to the definition file")
-	runCmd.Flags().StringVar(&runParams.ID, "id", "", "id of the resource to run")
+	runCmd.Flags().StringSliceVarP(&runParams.DefinitionFiles, "file", "f", []string{}, "path to the definition file (can be defined multiple times)")
+	runCmd.Flags().StringVarP(&runParams.ID, "id", "", "", "id of the resource to run (can be defined multiple times)")
```
Comment (danielbdias) [inline cli/cmd/resource_run_cmd.go:44]: I thought about having multiple IDs, but I reverted the changes and forgot to change here:

```suggestion
	runCmd.Flags().StringVarP(&runParams.ID, "id", "", "", "id of the resource to run")
```

File: cli/cmd/resource_run_cmd.go [hunk 2]
```diff
@@ -83,23 +56,37 @@ func init() {
...
+		JUnitOuptutFile: runParams.JUnitOuptutFile,
+		RequiredGates:   runParams.RequiredGates,
 	}
 
-	return "", fmt.Errorf("resourceName is empty and no definition file provided")
+	exitCode, err := orchestrator.Run(ctx, runParams, output)
+	ExitCLI(exitCode)
 
+	// ExitCLI will exit the process, so this return is just to satisfy the compiler
+	return "", err
```
Comment (danielbdias) [inline cli/cmd/resource_run_cmd.go:89]: I remember this comic every time we need to satisfy our Go compiler xD :
image

File: cli/formatters/multiple_runs.go
Comment (danielbdias) [inline cli/formatters/multiple_runs.go:1]: Since this formatter is only used on Cloud features, does it make sense to move it to the `cloud` package?

Comment (danielbdias) [reply cli/formatters/multiple_runs.go:1]: cc @mathnogueira

**Review by danielbdias — APPROVED** [review]

File: cli/formatters/multiple_runs.go
Comment (xoscar) [reply cli/formatters/multiple_runs.go:1]: Once we finish this feature, I'll create a cleanup PR with some changes, I believe we might not need the separation between cloud and OSS, even if someone wants to replicate the behavior, it is handled by the server side which wont be supported by the OSS server

File: cli/cmd/resource_run_cmd.go (hunk 1 above)
Comment (xoscar) [reply cli/cmd/resource_run_cmd.go:44]: If you don't mind, I'll fix this in the next PR, I want to have this merged 🥴

**Pull request merged**